doc/examples/pulsating-sphere-displacement.py

```diff
@@ -0,0 +1,72 @@
+"""
+Particle displacement of a pulsating sphere.
+"""
+import sfs
+import numpy as np
+from matplotlib import pyplot as plt
+from matplotlib import animation, patches
+
+# Pulsating sphere
+x0 = [0, 0, 0]  # position
+a = 0.25  # radius
+d_peak = 0.05  # peak value of the surface displacement
+f = 1000  # frequency
+omega = 2 * np.pi * f  # angular frequency
+ka = sfs.util.wavenumber(omega) * a
+
+# Temporal sampling for animation
+fs = f * 10  # sampling frequency
+L = int(np.round(fs / f))  # number of frames corresponding to one period
+t = np.arange(L) / fs  # time
+
+# Uniform grid
+N = 120
+xmin, xmax = -1, 1
+ymin, ymax = -1, 1
+x = np.tile(np.linspace(xmin, xmax, N), N)
+y = np.repeat(np.linspace(ymin, ymax, N), N)
+z = 0
+grid = sfs.util.XyzComponents([x, y, z])
+
+# Particle displacement
+d = sfs.mono.source.pulsating_sphere_displacement(omega, x0, a, d_peak, grid)
+
+# Animation
+fig = plt.figure(figsize=(8, 8))
```

Comment:
It would be great to put all the plotting stuff into a function if that's possible.

And make the whole file `import`able.

The concrete definitions could then go into such a section in the end of the file:

```python
if __name__ == '__main__':
    x0 = ...
    a = ...

    ...

    ani = ...
    ani.save(...)
```

Then the file could also be imported and it would be easier to play around with it.